doc/source/operations/rocky-linux-9.rst

```diff
@@ -0,0 +1,517 @@
+
+   .. code-block:: console
+
+      ceph orch ps <hostname>
+
+3. Deprovision the storage node:
+
+   .. code:: console
+
+      kayobe overcloud deprovision -l <hostname>
+
+4. Reprovision the storage node:
+
+   .. code:: console
+
+      kayobe overcloud provision -l <hostname>
+
+5. Host configure:
+
+   .. code-block:: console
+
+      kayobe overcloud host configure -l <hostname>
+
+6. Make sure the cephadm public key is in ``authorized_keys`` for stack or
+   root user - depends on your setup. For example, your SSH key may
+   already be defined in ``users.yml`` . If in doubt, run the cephadm
+   deploy playbook to copy the SSH key and install the cephadm binary.
+
+   .. code-block:: console
+
+      kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/cephadm-deploy.yml
+
+6. Take the host out of maintenance mode:
+
+   .. code-block:: console
+
+      ceph orch host maintenance exit <hostname>
+
+7. Make sure that everything is back in working condition before moving
+   on to the next host:
+
+   .. code-block:: console
+
+      ceph -s
+      ceph -w
+
+Seed
+====
+
+Potential issues
+----------------
+
+- The process depends a lot on the structure of the seed’s volumes. By
+  default two volumes are created (one root volume and one data
+  volume), however only the root volume is actually used. Most
+  deployments have this behaviour overridden so that both the volumes
+  are used and either ``/var/lib/docker`` or
+  ``/var/lib/docker/volumes`` is mounted to the data volume. This setup
+  makes it considerably easier to migrate the seed, as the root volume
+  can be deleted and the seed can be reprovisioned, leaving the data
+  volume intact throughout. If the deployment is using the default
+  setup, and nothing is stored in the data volume, the first step
+  should be to back up either the docker volumes or the entire docker
+  directory. This should then be restored to the seed after
+  ``seed host configure``
+- The mariadb process within the bifrost_deploy container needs to be
+  gracefully stopped. mariadb can’t boot a newer version if the
+  previous version stopped with an error.
+
+Full procedure
+--------------
+
+1.  On the seed, check the LVM configuration:
+
+    .. code:: console
+
+       lsblk
+
+2.  If the data volume is not mounted at either ``/var/lib/docker`` or
+    ``/var/lib/docker/volumes``, make an external copy of the data
+    somewhere on the seed hypervisor.
+
+3.  On the seed, stop the MariaDB process within the bifrost_deploy
+    container:
+
+    .. code:: console
+
+       sudo docker exec bifrost_deploy systemctl stop mariadb
+
+4.  On the seed, stop docker:
+
+    .. code:: console
+
+       sudo systemctl stop docker
+
+5.  On the seed, shut down the host:
+
+    .. code:: console
+
+       sudo systemctl poweroff
+
+6.  Wait for the VM to shut down:
+
+    .. code:: console
+
+       watch sudo virsh list --all
+
+7.  Back up the VM volumes on the seed hypervisor
```

Comment:
I've had a problem two times that upgrade of mariadb (because bifrost uses in-distro mariadb packages, so it's a database upgrade when you try to run with the same data) that it reported it cannot upgrade after a crash - what saved me was mysqldump...